project_task_stage_change_restriction/models/__init__.py

```
@@ -0,0 +1,5 @@
+# Copyright (C) 2025 Cetmix OÜ
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from . import project_task_stage
+from . import project_task
```

Flake8 (7.2.0)

[warning] no newline at end of file

(W292)